benchopt/tests/test_runner.py

```diff
@@ -323,8 +324,12 @@ def get_data(self):
     out.check_output("ValueError: Failing solver.", repetition=2)
 
 
+# Under windows, the function needs to be pickleable
+# for parallel jobs to work with joblib
 @pytest.mark.parametrize('n_jobs', [1, 2])
 def test_benchopt_run_script(n_jobs, no_debug_log):
+    if sys.platform == 'win32' and n_jobs > 1:
+        pytest.skip("Parallel jobs are incompatible with Windows")
```

Comment:
This test, added in the [following commit](https://github.com/benchopt/benchopt/commit/ace41416f1915ab7809c5b19c2f588e939bd6974#diff-cd2463708ecfd4cb8e59743e94eb96708f22b4662a4851c871635ee469223b4d), seems to fail on Windows because it needs [pickleable functions](https://github.com/joblib/joblib/issues/174#issuecomment-65410575). I've skipped it for now.